### DISCUSSION TITLE
Suggest connections when select a text

### DISCUSSION THREAD

**Discussion by nhanvu327** [athensresearch/athens#892] [Feature Requests]
As one of the core principles of the Zettelkasten method, the user should make connections between ideas as much as possible. So when a user selects a text, Athens should do a search to suggest the old blocks or pages for connections.

**Comment by nhanvu327** [comment]
do you agree ? @tangjeff0

**Comment by jefftangx** [comment]
I think auto-suggestions are a good idea, regardless if text is selected or not, though that could be a good trigger.

**Comment by nhanvu327** [comment]
cool, I think this feature is important. Should we add this to roadmap ?

**Comment by phrohdoh** [comment]
Have the performance implications of this been considered/discussed?
I highlight text _a lot_ and would rather not pay a price for a feature I _suspect_ I won't use (if it isn't forced on me, but maybe I will want/use the feature with time).

**Comment by Limezy** [comment]
When some text is selected, would also be very good to have an option for right clic > create a page, and that would put [[ ... ]] around the selection

**Reply by jefftangx** [reply]
Agreed
